app/views/shared/_alert.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="alert alert-danger alert-dismissible fade show" role="alert">
+    <%= alert %>
+    <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+</div>
```

Comment:
newline missing at the end of the file :)

Reply:
as always :(
by the way from c5ce5fa commit Github does not do quality tests. Maybe you have some idea why?

Reply:
magic! it works now! :)